README.md

```diff
@@ -307,6 +308,14 @@ They can be added to a scenario file like this:
         tasks = ["assemble"]
 
         apply-build-script-change-to = "build.gradle.kts"
+        apply-project-dependency-change-to {
+            files = ["build.gradle"]
+            # Default number of applied-projects-count is 3.
+            # Gradle-profiler will generate minimal number of projects to have a unique combination of project dependencies every run.
+            # Note: Number of projects is calculated as binomial coffiecient: "X choose applied-projects-count = iterations * files",
+            # due to that, higher the applied-projects-count is, less projects will be generated.
+            applied-projects-count = 3
```

Comment:
This term is still not meaningful for me. What is "applied" is the change itself, not the projects. What we want here is to say how many subproject dependencies the profiler is supposed to add. How about:

`project-dependency-count = 3`

or:

`dependency-count = 3`

or even simply:

`count = 3`

?

I think in the context even `count` makes perfect sense.

Reply:
Changed to dependency-count. For me it seems `count` does not hold enough information. And `dependency-count` is nice in case we will have also `apply-external-dependency-change-to` or smth similar in future